benchmarks/source-flotiq/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "gatsby-starter-hello-world",
```

Comment:
Probably worth updating this name and the description

Reply:
Done, thanks for pointing out.